### ISSUE TITLE
Can't create multi include projections for secondary index

### ISSUE THREAD

**Issue by Edwardsj** [guregu/dynamo#88]
createtable.go line 119
```
 if projection == IncludeProjection {
	for _, attr := range includeAttribs {
		proj.NonKeyAttributes = append(proj.NonKeyAttributes, &attr) // wrong usage
	}
}
```
NonKeyAttributes will always be the last one.
It will case `ValidationException: Cannot have two attributes with the same name`

**Comment by guregu** [comment]
Thanks for the report!
Can you give me some example code that makes this error happen? That way I can make sure it's fixed.

**Comment by Edwardsj** [comment]
sure, this is my example.
```
type ormAccount struct {
	ID         string    `json:"i" dynamo:"ID" index:"idxAccount0,hash" to:"true"`
	Name       string    `json:"n" dynamo:"Name,hash" to:"true"`
	PlatformID int64     `json:"pi" dynamo:"PlatformID,range" to:"true"`
	AddAt      time.Time `json:"aa" dynamo:"AddAt"`
	LastAuthAt time.Time `json:"laa" dynamo:"LastAuthAt"`
}

err := iAccount.conn.CreateTable(iAccount.name, ormAccount{}).
	Provision(5, 5).
	Project("idxAccount0", dynamo.IncludeProjection, "Name","PlatformID").
	ProvisionIndex("idxAccount0", 5, 5).
	Run()
fmt.Println(err)
```

**Comment by guregu** [comment]
I fixed this in 876421284dfa171a160a910227292b9091a01f44 and released v1.1.1
The code you posted seems to work for me now.
Let me know if you have any more problems! Thanks for the bug report.
